jupyterlab_gcedetails/src/details_widget.tsx

```diff
@@ -84,19 +83,14 @@ export class VmDetails extends React.Component<Props, State> {
         <span
           className={classes(STYLES.icon, ICON_CLASS, STYLES.interactiveHover)}
           title="Show all details"
-          onClick={() => this.showDialog()}
+          onClick={() => this.setState({ formDisplayed: true })}
         ></span>
         <WidgetPopup>
           <ResourceUtilizationCharts detailsServer={detailsServer} />
         </WidgetPopup>
         <span className={classes(STYLES.interactiveHover)}>
           {details ? this.getDisplayedDetails(details) : noDetailsMessage}
         </span>
-        <span
-          className={classes(STYLES.icon, ICON_CLASS, STYLES.interactiveHover)}
-          title="Show form"
-          onClick={() => this.setState({ formDisplayed: true })}
-        ></span>
         {formDisplayed && (
           <HardwareScalingDialog
```

Comment:
Nit but maybe we should change the name of the component from _HardwareScalingDialog_ and the state variable from _showForm_ to something more generic since the dialog now shows the hardware configuration details as well as allows you to modify them. Maybe something like _HardwareConfigurationDialog_ and _showDialog_?